### ISSUE TITLE
Embedded Links without https://

### ISSUE THREAD

**Issue by tobias-g1** [hapramp/alpha-web#77]
#### Expected behavior

If you don't include https:// within your embedded link when creating a new article a user should either be shown an error warning, it should automatically prepend https:// or https://. It should not render the 1ramp create a post page as a default link. 

#### Actual Behaviour

If you enter a link without https:// or http:// it will not show the link that you added, instead it will show the 1ramp create a post page within the embedded link.

#### Steps to reproduce

1. Go to https://alpha.1ramp.io/feed/
2. Sign in using your Steem Account
3. Click the floating menu option shown in the bottom right, select article
4. Click the embedded link option shown in the editor
5. Type in www.google.co.uk
6. Note that the embedded link, does load www.google.co.uk it loads the editor.
7. Repeat step 4 and type in https://www.google.co.uk
8. Note that it loads correctly. 

#### Environment

- Browser: Google Chrome (Version 65.0.3325.162 (Official Build) (64-bit))
- Device: MacBook Pro (Retina, 15-inch, Late 2013)
- Operating system: MacOS HighSierra Version 10.13

#### Recording of Issue

The following issue can be seen in the view below:

https://youtu.be/IwxxoiDYmDU

**Comment by singhpratyush** [comment]
Thanks for reporting @tobias-g1. There is certainly a wrong behavior here.

I think that the editor should support just a few types of embed links (like Youtube embed) as embedding links like https://www.google.co.uk won't make sense for a blog.

**Comment by tobias-g1** [comment]
I agree the google link was purely for demonstration. I think you could create a clear UX by pre-defining them. I haven't looked very deeply at the code, but I assume they're using iFrames at some point, pre-defining them would also help to avoid x-frame security issues too.
